types/msg.go

```diff
@@ -264,6 +265,14 @@ type WithdrawDelegatorRewardMsg struct {
 	Validator string `json:"validator"`
 }
 
+// FundCommunityPoolMsg is translated to a [MsgFundCommunityPool](https://github.com/cosmos/cosmos-sdk/blob/v0.42.4/proto/cosmos/distribution/v1beta1/tx.proto#LL69C1-L76C2).
+type FundCommunityPoolMsg struct {
+	// Amount is the list of coins to be send to the community pool
+	Amount     Coins  `json:"amount"`
+	// Depositor is the funding account
+	Depositor  string `json:"depositor"`
```

Comment:
This field should be removed. It is always the contract adddress. See e.g. SendMsg and BurnMsg above where the from address is also skipped.

Reply:
Ok. Thank you for quick feedback. Will do :)

Reply:
Ok. Adapted the Msg accordingly. Not sure why the ci audit breaks?!

Reply:
This is independent of your work. I'll take care of that. Thank you

Reply:
Alright. Thx.